plugin/mutate/source/dextool/plugin/mutate/backend/report/html/tmpl.d

```diff
@@ -76,8 +76,14 @@ Table tmplSortableTable(Element n, string[] header) @trusted {
     auto tr = base.div.parentDocument.createElement("tr");
     foreach (h; header.enumerate) {
         auto th = tr.addChild("th", h.value);
-        DashboardCss.sortableTableCol(th).setAttribute("id",
-                format!"col-%s"(h.index)).appendText(" ").addChild("i").addClass("right");
+        if (h.value == "Change") {
```

Comment:
tmplSortableTable is generic thus it can't have specific knowledge about how columns in other parts of dextool should be formatted. change tmplSortableTable to e.g. taking a callback as a parameter. then you can implement this logic at the callsite.